### ISSUE TITLE
Improve options for sending e-mails

### ISSUE THREAD

**Issue by apertome** [perfsonar/psconfig-web#48]
Currently the only way you can send e-mails is by configuring an SMTP server, user and password. I have not figured out a good service account to use to test this yet, and also I think many people may prefer to have it run its own `sendmail` or `postfix` service for doing this, which is what ... well, all the other perfSONAR pieces do, as far as I'm aware.

Some options here are running an MTA within the current docker container, or spinning up an additional Docker container that could act as an SMTP server.

This is kind of important, because right now the only way I have gotten account creation to work is by disabling confirmation of e-mail accounts, which is obviously not a practice we would recommend.

**Comment by apertome** [comment]
For the moment (4.1b1), this is fine, we will want to expand on e-mail config options in a future release, however.

**Comment by ShawnMcKee** [comment]
Hi Mark,
Yes, this is a real issue.  I wanted to add some new users to the WLCG/OSG instance and they are unable to register.   Can you describe how best to add them in the current PWA ?
Shawn

**Comment by apertome** [comment]
@ShawnMcKee I've got two options for how to do this now; one involves running an additional docker container with postfix running to send the e-mails out. The other involves configuring an external mail server.

Docs for this new process are here; not merged yet since I wanted to get some more eyes on it  first:

https://github.com/perfsonar/psconfig-web/tree/email-options-48#container-installation


TL;DR
in `/etc/pwa/auth/index.js`
set
```javascript
mailer: {
    host: 'postfix',
    secure: false,
    port: 25,
    tls: {
            // do not fail on invalid certs
            rejectUnauthorized: false
    }
}
```

and instantiate the docker postfix container as follows:
```docker run \
        --network pwa \
        -d --name postfix \
        -p 587:25 \
        --restart always \
        yorkshirekev/postfix HOSTNAME
```

Let me know your thoughts.

**Comment by epcjr** [comment]
Another micro-service!  LOL!

Later,

-Ed

**Comment by ShawnMcKee** [comment]
Thanks Michael,

I like the other container idea.   I assume your testing worked?   I guess
the only concern would be that there isn't a way for other to hijack the
SMTP server to send spam....

Let me know when you want me to test it.

Shawn

**Comment by apertome** [comment]
This needs a little review and then we can close it
